src/Mutator/Rounding/Round.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\Rounding;
+
+use Infection\Mutator\Util\Mutator;
+use PhpParser\Node;
+
+/**
+ * @internal
+ */
+final class Round extends Mutator
+{
+    /**
+     * Replaces "floor();" and "ceil();" with "round();"
+     *
+     * @param Node|Node\Name $node
+     *
+     * @return Node\Expr\Cast\Int_
```

Comment:
This method doesn't return an `Node\Expr\Cast\Int_`, how about changing it to just `Node` for now, or `Node|Node\Expr\FuncCall`